src/App.jsx

```diff
@@ -14,70 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        { (selectedGood.length > 0)
+          ? (
+            `${selectedGood} is selected`
+          )
+          : 'No goods selected'}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+        { (selectedGood)
```

Comment:
don't see any sense to a ternary operator here, because you have an empty space, as a part of this condition 
I recommend you replace it here 
{selectedGood => ? ...  :  ' ') to  {selectedGood => ...}
... -  it's a content